src/main/msp/msp_serial.h

```diff
@@ -39,23 +38,24 @@ typedef enum {
 } mspEvaluateNonMspData_e;
 
 #define MSP_PORT_INBUF_SIZE 64
```

Comment:
The serial protocol is identical. I only propose to limit maximum packet size to some reasonable value (possibly slightly larger than 64 bytes), both for command and response